dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetField.java

```diff
@@ -430,14 +430,15 @@ private String getFieldDisplayValue(String value, DatasetFieldType fieldType) {
             if (StringUtils.isBlank(format)) {
                 format = "#VALUE";
             }
-            String sanitizedValue = !fieldType.isSanitizeHtml() ?
-                    value :
-                    MarkupChecker.sanitizeBasicHTML(value);
+            if (FieldType.TEXTBOX.equals(fieldType.getFieldType())) {
+                value = value.replaceAll("\\n", "<br>");
```

Comment:
What will happen if we enter text "\n" into description?

Reply:
I've checked and nothing erroneous happens.

Reply:
When we will display such field that will contain text `\n` - we will display text `\n` or the extra line will be produced?

Reply:
The text ``\n`` would be displayed as ``\n`` (two characters).
The newline is implicitly converted to ``\n`` (one character) and will be displayed as newline.